### PR TITLE
[bugs] add check for LP: #1906280

### PR DESCRIPTION
**Pull request by freyes**
This patch implements a check to detect ocurrences of bug 1906280, this
will affect systems running systemd>=237-3ubuntu10.43 and
charm-neutron-openvswitch<21.01

Related-Bug: https://bugs.launchpad.net/charm-ovn-chassis/+bug/1906280

### CODE REVIEW

File: tests/unit/fake_data_root/sos_commands/dpkg/dpkg_-l [hunk 1]
```diff
@@ -947,7 +947,7 @@ ii  ssh-import-id                          5.5-0ubuntu1
 ii  strace                                 4.11-1ubuntu3                                   amd64        System call tracer
 ii  sudo                                   1.8.16-0ubuntu1.10                              amd64        Provide limited super user privileges to specific users
 ii  sysstat                                11.2.0-1ubuntu0.3                               amd64        system performance tools for Linux
-ii  systemd                                229-4ubuntu21.28                                amd64        system and service manager
+ii  systemd                                237-3ubuntu10.46                                amd64        system and service manager
 ii  systemd-sysv                           229-4ubuntu21.28                                amd64        system and service manager - SysV links
```
Comment (dosaboy) [inline tests/unit/fake_data_root/sos_commands/dpkg/dpkg_-l:951]: can you make all the systemd* packages the same version

**Review by dosaboy — CHANGES_REQUESTED** [review]
Hi Felipe, I would like to land this asap but its gonna need two small updates; first can you fix the systemd version (see comment) and it also needs a rebase since there are some imports that have changed name. Thanks.

File: tests/unit/fake_data_root/sos_commands/dpkg/dpkg_-l (hunk 1 above)
Comment (freyes) [reply tests/unit/fake_data_root/sos_commands/dpkg/dpkg_-l]: done.

**Comment by freyes** [comment]
> Hi Felipe, I would like to land this asap but its gonna need two small updates; first can you fix the systemd version (see comment) and it also needs a rebase since there are some imports that have changed name. Thanks.

Updated the fixture, rebased, resolved merge conflicts, tox is happy ;-)

**Review by dosaboy — CHANGES_REQUESTED** [review]
Hey freyes, I am hitting an error when i run a snap contaning this change:

-Traceback (most recent call last):
  File "/snap/hotsos/x1/plugins/openstack/parts/octavia_bugs.py", line 8, in <module>
    import git
ModuleNotFoundError: No module named 'git'

We either need to suppress that exception or update the snapcraft.yaml to install that dep

**Comment by freyes** [comment]
Added a fix for this issue, which it's basically make the snap to pull
the dependencies listed in the requirements.txt

**Comment by dosaboy** [comment]
closing for now. will revisit later.

**Pull request closed without merging**